app/views/hyrax/homepage/_recent_document.html.erb

```diff
@@ -1,16 +1,16 @@
-    <tr>
+    <tr class="recent-item">
       <td class="col-sm-2">
-        <%= link_to_profile recent_document.depositor("no depositor value") %>
         <%= link_to [main_app, recent_document] do %>
-          <%= render_thumbnail_tag recent_document, { width: 45 }, suppress_link: true %>
+          <%= render_thumbnail_tag recent_document, { width: 90 }, suppress_link: true %>
         <% end %>
       </td>
       <td>
-        <h3>
-          <span class="sr-only">Title</span><%= link_to truncate(recent_document.to_s, length: 28, separator: ' '), [main_app, recent_document] %>
-        </h3>
+        <span class="sr-only">Title</span><h3><%= link_to recent_document.to_s, [main_app, recent_document] %></h3>
         <p>
-          <span class="sr-only">Keywords</span><%= link_to_facet_list(recent_document.keyword, 'keyword', 'no keywords specified').html_safe %>
+          <span class="recent-label">Depositor:</span> <%= link_to_profile recent_document.depositor('no depositor value') %>
```

Comment:
How would you feel about including the bare strings in this file in i18n so they can be translated in the next release? (These strings may in fact already be translated...)